src/libs/ajax/DataRepo.ts

```diff
@@ -167,7 +167,7 @@ export interface DataRepoContract {
     getConcepts(parent: SnapshotBuilderConcept): Promise<GetConceptsResponse>;
     getConceptsHierarchy(concept: SnapshotBuilderConcept): Promise<GetConceptsHierarchyResponse>;
     // Search returns a list of matching concepts with a domain sorted by participant count. The result is truncated to N concepts.
-    searchConcepts(domain: SnapshotBuilderConcept, text: string): Promise<SearchConceptsResponse>;
+    searchConcepts(domain: SnapshotBuilderConcept, text?: string): Promise<SearchConceptsResponse>;
```

Comment:
With this change I think optional is no longer necessary here.
```suggestion
    searchConcepts(domain: SnapshotBuilderConcept, text: string): Promise<SearchConceptsResponse>;
```

Reply:
Good catch!